### DISCUSSION TITLE
Custom 404 error page

### DISCUSSION THREAD

**Discussion by ivanov-v** [vercel/vercel#4892] [Help] [answered]
I'm trying to define a custom 404 error page with the following routes in _vercel.json_. The page opens correctly, but the status is 200 instead of 404.

https://stm-labs-website.vercel.app/en/fdsf

`{
  "version": 2,
  "routes": [
    {
      "src": "/",
      "dest": "/api/localeRedirect.js"
    },
    {"handle": "filesystem"},
    {
      "src": "/(en|ru)/.*",
      "status": 404,
      "dest": "/$1/404/index.html"
    },
    {
      "src": "/.*",
      "status": 404,
      "dest": "/ru/404/index.html"
    }
  ]
}
`

**Comment by styfle** [comment, marked as answer]
I just checked and the link above returns 404 status code.

![image](https://user-images.githubusercontent.com/229881/88114810-bc168180-cb82-11ea-83c0-22c5ebab7da7.png)

Also note, you can usually avoid creating `routes` for custom 404 and instead emit a `404.html` in the output directory.

https://vercel.com/docs/configuration#routes/advanced/custom-404
